cmd/aterm/appdialogs/upload_prompt.go

```diff
@@ -11,15 +11,15 @@ import (
 	"strings"
 	"sync"
 
-	"github.com/pkg/errors"
 	"github.com/theparanoids/ashirt-server/backend/dtos"
 	"github.com/theparanoids/aterm/dialog"
+	"github.com/theparanoids/aterm/errors"
 	"github.com/theparanoids/aterm/fancy"
 	"github.com/theparanoids/aterm/network"
 )
 
-var errorCancelled = errors.New("Cancelled")
-var errorAlreadyExists = errors.New("Already Exists")
+var errorCancelled = fmt.Errorf("Cancelled")
```

Comment:
Naming convention in Go for naming errors is ErrCancelled, ErrAlreadyExists, etc. Typically they are exported because they are not not defined in the main package but can be private if not necessary. They're defined this way so that they can be checked later for specific types of errors

Reply:
Yeah, I don't know why I missed that. Should be fixed now.